src/client/views/rooms/list.html

```diff
@@ -29,7 +29,7 @@ <h2 class="header" ng-class="{ 'empty': rooms.isEmpty() }">
                 />
             </span>
             <form ng-submit="createRoom()" class="top-right-form">
-                <button type="submit" id="submit">
+                <button type="submit" id="submit" analytics-on="click" analytics-event="Create Room">
```

Comment:
Tracking "clicks" doesn't really make sens here: since you can create the room by pressing <kbd>Enter</kbd> in the text field. We need domain action oriented events.